docker/1.5.1/py3/Dockerfile.eia

```diff
@@ -0,0 +1,92 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.0.5
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.6.8
+ARG HEALTH_CHECK_VERSION=1.3.3
+
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    libopencv-dev \
+    openjdk-8-jdk-headless \
+    vim \
+    wget \
+    zlib1g-dev \
+ && rm -rf /var/lib/apt/lists/*
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib" \
+    PYTHONIOENCODING=UTF-8 \
+    LC_ALL=C.UTF-8
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xvf Python-$PYTHON_VERSION.tgz \
+ && cd Python-$PYTHON_VERSION \
+ && ./configure \
+ && make \
+ && make install \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    tk-dev \
+    libgdbm-dev \
+    libc6-dev \
+    libbz2-dev \
+ && make \
+ && make install \
+ && rm -rf ../Python-$PYTHON_VERSION* \
+ && ln -s /usr/local/bin/pip3 /usr/bin/pip \
+ && ln -s $(which ${PYTHON}) /usr/local/bin/python
+
+WORKDIR /
+
+RUN wget https://amazonei-healthcheck.s3.amazonaws.com/v${HEALTH_CHECK_VERSION}/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -O /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
+ && tar -xvf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -C /opt/ \
+ && rm -rf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
+ && chmod a+x /opt/ei_health_check/bin/health_check
+
+RUN pip install --no-cache-dir --upgrade \
+    pip \
+    setuptools
+
+RUN pip install --no-cache-dir \
+    https://s3.amazonaws.com/amazonei-apachemxnet/amazonei_mxnet-1.5.1-py2.py3-none-manylinux1_x86_64.whl \
+    mxnet-model-server==$MMS_VERSION \
+    keras-mxnet==2.2.4.1 \
+    onnx==1.4.1 \
+    # use sagemaker-inference version comaptible with MMS_VERSION=1.0.5
+    sagemaker-inference==1.1.0 \
```

Comment:
Do we need Numpy package. if yes, we can have numpy==1.17.4. If it passes the safety checks.

Reply:
Looks like the 1.4.1 EI dockerfile did not have numpy. Is there a reason to add it here?

Reply:
It is present in Pytorch EIA and MXNet Cpu/GPU. So I though it would be nice package to have it.

Reply:
Sure, add it.